doc/Performing_IO_Operations_Using_S3Client.rst

```diff
@@ -4,6 +4,12 @@ Performing IO Operations Using S3 Client
 
 If you want to use the provided OVA to test CORTX and observe activity in its GUI, you should have already followed the `instructions <CORTX_on_Open_Virtual_Appliance.rst>`_ to install and configure the OVA.  If you have not yet done so, please do that before following the rest of the instructions on this page.
 
+To perform the IO operations in CORTX, you need to install the AWS CLI. You can install the AWS CLI either on the same OVA or you can create a new VM and install it.
+
+For more information on:
+
+-  AWS CLI on the same OVA, see `AWS CLI on the CORTX OVA VM <#AWS-CLI-on-the-CORTX-OVA-VM>`__.
+-  AWS CLI on the new VM, see `AWS CLI on a different VM <#AWS-CLI-on-a-different-VM>`__.
 
```

Comment:
's3client' should be mentioned as "AWS CLI on same OVA as client"

Reply:
As discussed, this is not required.